src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.ts

```diff
@@ -426,10 +428,13 @@ export class OsdListComponent extends ListWithDetails implements OnInit {
     }
   }
 
-  getOsdList() {
-    const observables = [this.osdService.getList(), this.osdService.getFlags()];
-    observableForkJoin(observables).subscribe((resp: [any[], string[]]) => {
-      this.osds = resp[0].map((osd) => {
+  getOsdList(context?: CdTableFetchDataContext) {
+    if (!context) context = new CdTableFetchDataContext(() => undefined);
+    const pagination_obs = this.osdService.getList(context?.toParams())
+    const observables = [pagination_obs.observable, this.osdService.getFlags()];
```

Comment:
Can't do it. We get the table ccontext for this component within this file because that's where the table is rendered. And we need it increase/decrease the limit of items in a request.